swfo/brdf_fallback.py

```diff
@@ -858,31 +831,41 @@ def write_brdf_fallback(brdf_dir, outdir, tile, year_from, year_to, filter_size,
     
     # remove doy 366 from further processing, final results for 366 will be symlinked to doy 365
     set_doys.remove(366)
-
+    
     with tempfile.TemporaryDirectory() as tmp_dir:
+        
         for band in BAND_LIST:
             write_brdf_fallback_band(h5_info, tile, band, tmp_dir, filter_size, set_doys,
                                      pthresh=10.0, data_chunks=(240, 240), compute_chunks=(240, 240),
                                      nprocs=nprocs, compression=compression)
-     
+        
+        start_ds, *_, end_ds = sorted(h5_info.keys())
+        tile_metadata = munge_metadata(h5_info[start_ds], start_ds, end_ds, only_id=False)
+        
+        with Pool(processes=nprocs) as pool: 
+            ids_brdf = pool.starmap(munge_metadata, [(fp, start_ds, end_ds) for key, fp in h5_info.items()])
+        
+        tile_metadata['lineage']['brdf_threshold'] = ids_brdf
+        
         with Pool(processes=nprocs) as pool: 
             pool.starmap(concatenate_files, [([str(fp) for fp in Path(tmp_dir).rglob(BRDF_MATCH_PATTERN
                                                                                      .format(tile, doy))],
                                               os.path.join(outdir, BRDF_AVG_FILE_FMT.format(tile, doy)),
-                                              h5_info, doy) for doy in set_doys])
+                                              h5_info, doy, tile_metadata) for doy in set_doys])
     
     # symlink doy 366 to the final results of doy 365 results 
-    os.symlink(os.path.join(outdir, BRDF_AVG_FILE_FMT.format(tile, 365)), 
-               os.path.join(outdir, BRDF_AVG_FILE_FMT.format(tile, 366)))
+    os.symlink(pjoin(outdir, BRDF_AVG_FILE_FMT.format(tile, 365)), 
+               pjoin(outdir, BRDF_AVG_FILE_FMT.format(tile, 366)))
+
 
 @click.command()
 @click.option('--brdf-dir', default='/g/data/v10/eoancillarydata.reS/brdf.av/MCD43A1.006/')
-@click.option('--outdir', default='/g/data/u46/users/pd1813/BRDF_PARAM/test_struct')
+@click.option('--outdir', default='/g/data/u46/users/pd1813/BRDF_PARAM/test_v32')
```

Comment:
feels odd to have publicly available code having hardcoded paths that outsiders do not have access to